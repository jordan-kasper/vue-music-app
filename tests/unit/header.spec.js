/* eslint-disable no-console */
/* eslint-disable no-undef */

import { shallowMount } from '@vue/test-utils';
import Header from '../../src/components/header.vue';

describe('Counter', () => {
  // Now mount the component and you have the wrapper
  const wrapper = shallowMount(Header);

  // it's also easy to check for the existence of elements
  it('Header loaded', () => {
    expect(wrapper.name()).toBe('Header');
  });
});
