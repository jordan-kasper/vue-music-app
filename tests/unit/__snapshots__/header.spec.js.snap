// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Header loaded 1`] = `
<nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark"><a href="#" class="navbar-brand">Music Search App</a> <button type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation" class="navbar-toggler"><span class="navbar-toggler-icon"></span></button>
  <div id="navbarsExampleDefault" class="collapse navbar-collapse">
    <ul class="navbar-nav mr-auto"></ul>
  </div>
</nav>
`;
